modules/blocks/process.js

```diff
@@ -447,6 +447,7 @@ __private.receiveForkOne = function (block, lastBlock, cb) {
 	// Keep the oldest block, or if both have same age, keep block with lower id
 	if (block.timestamp > lastBlock.timestamp || (block.timestamp === lastBlock.timestamp && block.id > lastBlock.id)) {
 		library.logger.info('Last block stands');
+		modules.transport.broadcastBlock({broadhash: modules.system.getBroadhash(), unmatchBroadhash: true}, lastBlock);
```

Comment:
I don't get it: why do we need to broadcast a block that was previously broadcasted (is a last block, therefore it was broadcasted by `applyBlock`)

Reply:
We want to update with the correct (from our point of view) block peers with different broadhash - the part of the network on a possible fork.

Reply:
Please add a comment to describe resulting broadcast.